includes/discovery/mef-evc.inc.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * Try to discover any MEF Links
+ */
+
+if ($device['os'] == 'coriant') {
```

Comment:
If it's a standard that devices should support then the if check can be completely removed as the module will only be on for relevant devices anyway.